android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireViewModel.kt

```diff
@@ -205,6 +205,12 @@ constructor(
           context = context,
         )
 
+      if (questionnaireConfig.setPractitionerDetails) {
+        bundle.entry.forEach { bundleEntry ->
+          bundleEntry.resource.appendPractitionerInfo(practitionerId)
+        }
+      }
```

Comment:
I do not think this is needed anymore. Can you confirm that the ResourceExtension function is used in the function that applies metadata on the resources?